src/main/java/com/ahaidychuk/dependencyversionupdater/model/WebHookPayload.java

```diff
@@ -0,0 +1,20 @@
+package com.ahaidychuk.dependencyversionupdater.model;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * Truncated payload of Git Push event for repository.
+ */
+@Data
+public class WebHookPayload {
+
+  private String after;   // commit Hash
+  private List<Commit> commits;
```

Comment:
Finals on method params is not bad, but how about having immutable model classes?

Reply:
Marked WebHook and Commit as final classes